src/style.css

```diff
@@ -0,0 +1,57 @@
+body {
+  margin: 0;
+}
+
+.grid-container {
+  display: grid;
+  grid-template-rows: repeat(6, 300px);
```

Comment:
it's not necessary to specify number of the rows in this case, you can use auto-fill or just leave only styles for columns